modules/admin_manual/pages/enterprise/metrics/metrics.adoc

```diff
@@ -0,0 +1,101 @@
+
+The Metrics data are provided as snapshot values in the JSON format and are optimized to be consumed by professional data analyzers (like Splunk, ELK or Prometheus/Grafana) to collect statistics, derive visualizations and to set alerts for certain events of interest. They can be perfectly combined with the ownCloud Audit Logs (provided by the https://marketplace.owncloud.com/apps/admin_audit[Auditing App]) to gather time series data and to create a monitoring and reporting engine for ownCloud.
+
+Specifically, the extension adds:
+- an API endpoint which allows querying snapshot values of the system data as well as per-user data
+- an API endpoint for downloading the data in the CSV format
+- a dashboard that displays the snapshot data in the ownCloud Web UI and offers a CSV download (accessible by ownCloud administrators via the app launcher)
+
+TIP: Please note: If you're operating very large instances (users/files/shares,) it is recommended to use a special setup in order to not put the production database under huge load when gathering the values. For this, please replicate your installation (application server + read-only database) and install/use the Metrics App on the replica.
+
+The following data is available:
+
+**System data**
+- Date/Time stamp - Server time of the request
+- Storage
+  - Used storage (this also includes storage for avatars and thumbnails)
+  - Free storage
+  - Total storage (used + free)
+  - Number of files
+- Number of users
+  - registered (total number of known users)
+  - active (number of users with `lastLogin` less than two weeks ago)
+  - concurrent (number of users with at least one active session)
+- Shares
+  - Number of user shares
+  - Number of group shares
+  - Number of guest shares
+  - Number of link shares
+  - Number of federated shares
+
+**Per-user data**
+- User ID
+- Display name
+- User backend
+- Last login
+- Active sessions
+- Quota
+  - Quota limit
+  - Quota usage
+- Number of files
+- Shares
+  - Number of user shares
+  - Number of group shares
+  - Number of guest shares
+  - Number of link shares
+  - Number of federated shares
+
+== Usage
+
+=== Authorization
+
+To get set up, you have to set a secret for authorization. This token is stored in config.php as `metrics_shared_secret` and needs to be sent with each request.
+
+See the following example on how to set it:
+
+```
+occ config:system:set "metrics_shared_secret" --value <your-metrics-secret>
+```
+
+TIP: You have to set the Metrics secret to use the dashboard.
+
+=== Endpoints and parameters
+
+==== Metrics endpoint
+
+To query for the Metrics data, use the following endpoint:
+
+```
+https://<your owncloud>/ocs/v1.php/apps/metrics/api/v1/metrics
```

Comment:
```suggestion
\https://<your owncloud>/ocs/v1.php/apps/metrics/api/v1/metrics
```
This will make the url not a link and therfore not an issue with broken links